test/test_bin.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pytest
+
+from make_iter_bins import FW_bins, Div_bins
+
+def test_wetted_fraction():
+    """Tests that wetted fraction is correctly computed 
+    for FW sub-bins.
+    """""
+    # first test FW Bin with 3 sub-bins
+    fw_bin = FW_bins.get_bin(8)
+    for sub_bin in fw_bin.sub_bins:
+        if sub_bin.mode == "shadowed":
+            shadowed_frac = sub_bin.wetted_frac
+        elif sub_bin.mode == "low_wetted":
+            low_wet_frac = sub_bin.wetted_frac
+        elif sub_bin.mode == "high_wetted":
+            high_wet_frac = sub_bin.wetted_frac
+
+    # 0.41066704,18.184414811440848,4.86154584,0.01654854884176685
+    expected_shadowed = 0.0
+    expected_low_wet = 0.01654854884176685 * 18.184414811440848 / 0.41066704
+    expected_high_wet = (1-0.01654854884176685) * 18.184414811440848 / 4.86154584
```

Comment:
No idea what I'm looking at here. I think it's data read from Wetted_Frac_bin_Data.csv?? If you want to test the functionality behind `read_wetted_data`, then you should probably:
1) make dummy bins,
2) make dummy wetted data and store it as csv
3) call `read_wetted_data`
4) test

Here, as soon as you csv file changes, you test will break, even though the functionality hasn't been touched